docs/en-US/component/autocomplete.md

```diff
@@ -41,35 +41,29 @@ autocomplete/remote-search
-| fit-input-width                   | whether the width of the dropdown is the same as the input                                                                 | <BooleanType />                                                                                | false        |
-
-```
+| Name                              | Description                                                                                                                | Type                                                                                      | Default      |
+| --------------------------------- | -------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ------------ |
+| placeholder                       | the placeholder of Autocomplete                                                                                            | ^[string]                                                                                 | —            |
+| clearable                         | whether to show clear button                                                                                               | ^[boolean]                                                                                | false        |
+| disabled                          | whether Autocomplete is disabled                                                                                           | ^[boolean]                                                                                | false        |
+| value-key                         | key name of the input suggestion object for display                                                                        | ^[string]                                                                                 | value        |
+| model-value / v-model             | binding value                                                                                                              | ^[string]                                                                                 | —            |
+| debounce                          | debounce delay when typing, in milliseconds                                                                                | ^[number]                                                                                 | 300          |
+| placement                         | placement of the popup menu                                                                                                | ^[enum]`'top' \| 'top- start' \| 'top-end' \| 'bottom' \| 'bottom-start' \| 'bottom-end'` | bottom-start |
+| fetch-suggestions                 | a method to fetch input suggestions. When suggestions are ready, invoke `callback(data:[])` to return them to Autocomplete | ^[Function]`(queryString': string, callback: callbackfn) => void`                         | —            |
```

Comment:
There seems to be an extra quotation mark
`(queryString': string, callback: callbackfn) => void`  => `(queryString: string, callback: callbackfn) => void`

Reply:
You are right!